### PR TITLE
Hello Microverse Project

### PR DESCRIPTION
**Pull request by dielupovissa**
# Changes made so far:
- Added a readme file
- Added a gitgnore file
- Added an HTML/CSS file
- Added a text file: it was set up to be ignored by git

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,31 @@
 # Hello-Microverse
 This is a hello microverse project for my course in Microverse.
+In this project, you will set up a "Hello world" repository. 
+No complex coding is required for this exercise.
```
Comment (RolandM99) [inline README.md]: - [ ]  Your README looks great 👍🏻  but I think you can improve it by making it more professional. A professional README file should have a project title, project description, list of technology used in the project, and instructions to set the project up locally. please refer to this [template](https://github.com/microverseinc/readme-template), you can copy and paste the content but remember to customize it to your project.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,31 @@
...
 This is a hello microverse project for my course in Microverse.
+In this project, you will set up a "Hello world" repository. 
+No complex coding is required for this exercise. 
+Your goal here is to master all of the tools and best practices you learned about in previous steps. 
+You will be using them in all Microverse projects and most likely in your future job as well, so it is important to know them!
+
+## Build With
+- HTML/CSS
+- Linter: github actions
+## General Requirements
```
Comment (RolandM99) [inline README.md:12]: Great Job by seting up  Github actions for your project. Unfortunally, all Linters checks have failed. It seems that you didn't set up them correctly.
image
To fix these issues, Kindly follow correctly the steps listed in this [documentation](https://github.com/microverseinc/linters-config/tree/master/html-css)

**Review by RolandM99 — CHANGES_REQUESTED** [review]
Hi @dielupovissa  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


### To highlight 💯 
- Follow Github flow ✔️ 

## Required Changes ♻️


- [ ] Make sure that your Pull Request has a descriptive title and tells us what changes are inside. Please, kindly fix this issue by editing your Pull Request. 
- [ ] You don't need to create for each requirement in this project a new branch, kindly add all requirements(HTML file, CSS file, test.md file,...) in this feature branch so that we can see all features you implemented for this milestone.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on @Rolandm99 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (sja-thedude) [reply README.md]: Good job implementing the changes asked by the previous reviewer. 👍 
Your readme is perfect!

File: README.md (hunk 2 above)
Comment (sja-thedude) [reply README.md:12]: All linters are passing! Great job! 👍

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @dielupovissa,  This is @sja-thedude your code reviewer.

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## Required: 

- [ ] Kindly replace your Pull request title with `Hello Microverse Project` which will make it clear what you are working on.


 ## Highlights
- All linters are passing 👍 
- Professional Readme 👍 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by dielupovissa** [comment]
I did the change of the title of pull reques as you mention @sja-thedude is there anything alse to change?

**Comment by sja-thedude** [comment]
No @dielupovissa, everything is okay. You can submit it for review.

**Comment by dielupovissa** [comment]
thanks

**Comment by dielupovissa** [comment]
I cannot submit it!

**Comment by dielupovissa** [comment]
@sja-thedude is there yet a review pending.

**Review by ShadyShawkat — APPROVED** [review]
## Hi @dielupovissa, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by dielupovissa** [comment]
Thank you! What link I have to submit @ShadyShawkat ?

**Comment by ShadyShawkat** [comment]
Hey @dielupovissa, you can submit the link to the repo after the approval for your project.

**Pull request merged**